pennylane/templates/subroutines/qpe.py

```diff
@@ -18,9 +18,10 @@
 import pennylane as qml
 from pennylane.queuing import QueuingManager
 from pennylane.operation import AnyWires, Operation, Operator
+from pennylane.resource import ErrorOperation
 
 
-class QuantumPhaseEstimation(Operation):
+class QuantumPhaseEstimation(ErrorOperation, Operation):
```

Comment:
Does it need to inherit from both `ErrorOperation` _and_ `Operation`?

Reply:
Just one